### ISSUE TITLE
High variation output in successive  trainings

### ISSUE THREAD

**Issue by ricardovalquin** [pybrain/pybrain#178]
Hi, I'm making an error surface [100X100] for choose the right parameters for the learning rate and the momentum for to make a forecasting  of a time series, the issue is that  when i run the iterations for the error surface i get errors values like 130, 80, 70 and some other times i get errors like 17, 10, 5, 6. is very wear i only have the training function into two fors cycles to do the mesh, like that:

for i in range(0, 100, 1):
    for j in range(0, 100, 1):
        trainer = BackpropTrainer(net, ds, learningrate = learningrate_v[i], momentum = momentum_v[j])
        print "training until the Convergence: ", trainer.trainUntilConvergence(verbose=True, trainingData=trainingSet, validationData=ds, maxEpochs=100)
        y = net.activate([normalized_data[-5], normalized_data[-4], normalized_data[-3]])   

can anyone bring me an idea about what could happen????
